### DISCUSSION TITLE
Log4J Vulnerabilities

### DISCUSSION THREAD

**Discussion by joshmurari** [actions/virtual-environments#4758] [Installed software]
Hey everyone,

Based on the recent public reports of vulnerabilities regarding Log4j, is this something that impacts the windows, ubuntu, or mac OS images that are built here?

**Comment by miketimofeev** [comment]
Hi @joshmurari!
We do not install log4j explicitly and have also inspected build logs and see no trace of log4j in any dependencies.

**Reply by joshmurari** [reply]
Thanks for the response!

**Reply by mthoger** [reply]
@miketimofeev Maven has log4j files, in the apaches logging paths. 
isn't that a part of the hole deal ?

**Reply by miketimofeev** [reply]
@mthoger it's just a cache that doesn't seem to be used anywhere. I have prepared the PR to clean up this directory just in case https://github.com/actions/virtual-environments/pull/4761

**Reply by mthoger** [reply]
Thanks for the response!

**Comment by spyder007** [comment]
@miketimofeev, for your information, a newly provisioned machine (as of 12/18) has log4j in that same cache, but the version is now 2.16 (a build from Sept 29 had 2.11).

It would seem that potentially Maven/Apache has updated their packages as of last week.

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
